.github/workflows/kuksa_databroker-cli_build.yml

```diff
@@ -207,15 +218,24 @@ jobs:
             ~/.cargo/registry/cache/
             ~/.cargo/git/db/
             target/
-          key: databroker-cli-bom-${{ hashFiles('**/Cargo.lock') }}
-
-      - name: "Createbom: License check and Dash output generation"
-        working-directory: ${{github.workspace}}/createbom
+          key: databroker-bom-${{ hashFiles('**/Cargo.lock') }}
+
+      # Follows the pattern from
+      # https://github.com/eclipse/dash-licenses?tab=readme-ov-file#example-rustcargo
+      - name: "Using cargo to create Dash input"
+        working-directory: ${{github.workspace}}/
+        # target all is not really needed, and will also return i.e. wasm deps, however
+        # better safe than sorry, the alternative would be running this for each currently
+        # built target  and combining the lists, but that would need adapting, when
+        # adding targets, or also when i.e. switching between MUSL/glibc. So this is safer
         run: |
-         which cargo-license || cargo install cargo-license
-         python3 createbom.py --dash ${{github.workspace}}/dash-databroker-cli ../databroker
+          cargo tree -e normal --prefix none --no-dedupe -p databroker-cli --target all --all-features  > ${{github.workspace}}/cargodeps
+          cat ${{github.workspace}}/cargodeps | sort -u  \
+          | grep -v '^[[:space:]]*$' | grep -v kuksa | grep -v databroker  \
+          | sed -E 's|([^ ]+) v([^ ]+).*|crate/cratesio/-/\1/\2|' \
+          > ${{github.workspace}}/dash-databroker-deps
 
       - name: Dash license check
         uses: eclipse-kuksa/kuksa-actions/check-dash@3
         with:
-          dashinput: ${{github.workspace}}/dash-databroker-cli
+          dashinput: ${{github.workspace}}/dash-databroker-deps
```

Comment:
The name (`dash-databroker-deps`) must be different than the name used for databroker, otherwise aggregated builds (like create draft release) will end up  with an error:

`Error: Failed to CreateArtifact: Received non-retryable error: Failed request: (409) Conflict: an artifact with this name already exists on the workflow run`


... as then both jobs use the same upload area.

If you change here (and on line 236) to `dash-databroker-cli-deps` then there should be no name conflict.

Test build at https://github.com/boschglobal/kuksa-databroker/actions/runs/8965789017

Reply:
Fixed